schemas/links/embeddedlinkend.json

```
@@ -0,0 +1,36 @@
+{
+  "link_type": {
+    "type": "string",
+    "enum": [
+      "END"
+    ]
+  },
+  "timestamp": {
+    "type": "string",
+    "format": "date-time"
+  },
+  "from": {
+    "description": "When consuming a CDEvent, you are consuming a parent event. So, when looking at the 'from' key, this is the parent's parent.",
+    "type": "object",
+    "properties": {
+      "context_id": {
+        "type": "string",
+        "minLength": 1
+      }
+    },
+    "required": [
+      "context_id"
+    ]
+  },
+  "tags": {
+    "type": "object",
+    "additionalProperties": true
+  },
+  "additionalProperties": false,
+  "type": "object",
+  "required": [
+    "link_type",
+    "timestamp"
```

The timestamp should not be here (see earlier comment). But I assume `from` should be required instead?
```suggestion
    "from"
```